sequoia/utils/readme.py

```diff
@@ -54,12 +54,11 @@ def get_tree_string(
             f"somehow."
         )
     setting: Type["Setting"] = root_setting
-    prefix: str = ""
```

Comment:
How did these changes get here?

Reply:
I'm not sure, it must have been some accident. Now that I use git cli and add my files manually it shouldn't happen again :)

Reply:
You'll have to get rid of these by rebasing your branch off of master. I can help you with that if you want.

Reply:
You'll have to get rid of these by rebasing your branch off of master. I can help you with that if you want.